### ISSUE TITLE
Sonar will not recognize vst dll

### ISSUE THREAD

**Issue by GoogleCodeExporter** [shaffer420/midi-shape-shifter#27]
```
What steps will reproduce the problem?
1.Install MSS
2.Scan vst plugins in Sonar X2

3.

What is the expected output?
I expect Sonar to recognize MSS


What is the actual output?
Sonar gets stuck when it finds MSS dll file


What version of the product are you using? On what operating system?
1.0.7 Beta

Please provide any additional information below.


```

Original issue reported on code.google.com by `brianlit...@gmail.com` on 19 Mar 2013 at 2:55

**Comment by GoogleCodeExporter** [comment]
```
Thanks for submitting this! I'm traveling right now but I'll try to test out 
MSS in Sonar when I get back (in about a week).
```

Original comment by `RobIan...@gmail.com` on 20 Mar 2013 at 9:34

**Comment by GoogleCodeExporter** [comment]
```
Are you using the 32-bit or 64-bit version of Sonar?
```

Original comment by `RobIan...@gmail.com` on 7 Apr 2013 at 9:08

**Comment by GoogleCodeExporter** [comment]
```
Thanks for getting back to me. 
I'm using the 64 bit version of Sonar x2
```

Original comment by `brianlit...@gmail.com` on 8 Apr 2013 at 12:03

**Comment by GoogleCodeExporter** [comment]
```
Ah ok. I just released the 64-bit version of MSS so you can get it from the 
downloads page now. Your Sonar should be able to recognize that version of MSS. 

When I run MSS inside of Sonar I'm getting an error message but I think that is 
an unrelated issue so let me know if this works for you. Thanks.
```

Original comment by `RobIan...@gmail.com` on 8 Apr 2013 at 4:03

**Comment by GoogleCodeExporter** [comment]
```
There is a new installer posted that should fix this issue (v1.0.7b)
```

Original comment by `RobIan...@gmail.com` on 19 Apr 2013 at 10:16
- Changed state: **Fixed**
